src/views/SearchView.vue

```diff
@@ -36,13 +37,22 @@ export default {
       store: useStore(),
     };
   },
+  data() {
+    return {
+      error: "",
+    };
+  },
   methods: {
     async searchForConcerts() {
+      this.error = "";
```

Comment:
wieso musst du das hier nochmal als leeren String initialisieren? Hast du ja schon in deinem data Prop.